lib/collections/collections.js

```diff
@@ -173,3 +173,10 @@ Themes.attachSchema(Schemas.Themes);
 export const Translations = new Mongo.Collection("Translations");
 
 Translations.attachSchema(Schemas.Translation);
+
+/**
+* Translations Collection
+*/
+export const Wallets = new Mongo.Collection("Wallets");
+
+Translations.attachSchema(Schemas.Wallets);
```

Comment:
@andela-pbirir this line here could be the reason for the collections mixup on mLab and the deployment, change it to `Wallets.attachSchema(Schemas.Wallets);` and see if it helps.